### DISCUSSION TITLE
通过 iStoreOS 中 Docker 安装的 1Panel中的防火墙服务问题

### DISCUSSION THREAD

**Discussion by IJayHuI** [istoreos/istoreos#1849] [Q&A]
IStoreOS基于openWrt，openWrt使用opkg作为包管理器。查看官网的教程是使用yum或者apt安装firewalld或者ufw。不知道openWrt这种怎么处理。
![屏幕截图 2024-10-12 224952](https://github.com/user-attachments/assets/57a57157-3104-44e6-910e-bb62956a8622)

**Comment by w9315273** [comment]
[这里提问](https://github.com/openwrt/openwrt/issues)
